rustfmt-core/rustfmt-lib/src/expr.rs

```diff
@@ -2035,21 +2041,22 @@ pub(crate) fn prefer_next_line(
 
 fn rewrite_expr_addrof(
     context: &RewriteContext<'_>,
+    _borrow_kind: ast::BorrowKind,
```

Comment:
Could you add an implementation for a raw reference operator, or create an issue for this so we don't forget? cc https://github.com/rust-lang/rust/issues/64490

Reply:
Sure! Have opened #4024 to keep track of it. I can work on implementing that too in a follow up if no one beats me to it